Warning in build on Arch Linux w/ gcc 5.3.0

Awesome library. Nice and clean pure C code!

However, when compiling the tests, I get following warning:
In file included from /usr/include/stdlib.h:24:0,
                 from tests.c:12:
/usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"

I had to remove -Werror from the CFLAGS to make it compile.

--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 # published by Sam Hocevar. See the COPYING file for more details.
 
 CFLAGS = -g -Wall -Wextra -Werror -std=c99
-CFLAGS += -D_POSIX_C_SOURCE=199309L -D_BSD_SOURCE -D_DARWIN_C_SOURCE
+CFLAGS += -D_POSIX_C_SOURCE=199309L -D_DEFAULT_SOURCE -D_DARWIN_C_SOURCE
 LDLIBS = -lcheck
 
 all: scan-build test example

--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 # published by Sam Hocevar. See the COPYING file for more details.
 
 CFLAGS = -g -Wall -Wextra -Werror -std=c99
-CFLAGS += -D_POSIX_C_SOURCE=199309L -D_DEFAULT_SOURCE -D_DARWIN_C_SOURCE
+CFLAGS += -D_POSIX_C_SOURCE=199309L -D_BSD_SOURCE -D_DEFAULT_SOURCE -D_DARWIN_C_SOURCE
 LDLIBS = -lcheck
 
 all: scan-build test example

--- a/minmea.c
+++ b/minmea.c
@@ -12,6 +12,7 @@
 #include <string.h>
 #include <ctype.h>
 #include <stdarg.h>
+#include <time.h>
 
 #define boolstr(s) ((s) ? "true" : "false")
 

